### ISSUE TITLE
Confusing default text when no alt text is provided

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#268]
Len Burns reported this issue. Basically, our alt tag over-ride language is confusing. I'm setting this as Critical, and assigning to Teresa to evaluate what this would take. And would we need to change the code for the default and also retroactively change the existing images, or is that the same thing? Pending that, we'll probably want to switch this to Could Wait.

See: http://admissions-dev.ucsc.edu/campus-life/index.html
Len's text:
The “No image alternative provided” leaves the user without a means of
knowing that the caption below it is associated with this photograph.  I
would suggest something like: “Photo, caption follows.”
